pkg/schema/definitions/fixtures_test.go

```diff
@@ -398,3 +400,114 @@ definitions:
       kind: "ConfigMap"
       version: "v1"
 `
+
+var (
+	rawSchemalessCRDs = `
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: schemaless.management.cattle.io
+spec:
+  conversion:
+    strategy: None
+  group: management.cattle.io
+  names:
+    kind: Schemaless
+    listKind: SchemalessList
+    plural: schemalese
+    singular: schemaless
+  scope: Cluster
+  versions:
+  - name: v2
+    schema:
+      openAPIV3Schema:
+        type: object
+        properties:
+          spec:
+            type: object
+            properties:
+              rkeConfig:
+                type: object
+                nullable: true
+        x-kubernetes-preserve-unknown-fields: true
+    served: true
+    storage: true
```

Comment:
From the issue, I don't think that this was the use case that was causing our problems. I think we need something with an empty openapiV3 schema. So like the below:
```suggestion
apiVersion: apiextensions.k8s.io/v1
kind: CustomResourceDefinition
metadata:
  name: schemaless.management.cattle.io
spec:
  conversion:
    strategy: None
  group: management.cattle.io
  names:
    kind: Schemaless
    listKind: SchemalessList
    plural: schemalese
    singular: schemaless
  scope: Cluster
  preserveUnkownFields: true
  versions:
  - name: v2
    served: true
    storage: true
```
Would you mind updating this and the models to be a bit more like what we could see in production?